k256/src/arithmetic.rs

```diff
@@ -33,10 +35,77 @@ pub(crate) const CURVE_EQUATION_B: FieldElement = FieldElement::from_bytes_unche
     0, 0, 0, 0, 0, 0, 0, CURVE_EQUATION_B_SINGLE as u8,
 ]);
 
+impl ToAffineBatch for Secp256k1 {
```

Comment:
I know this code isn't DRY, but the call to `invert_batch_generic` in one case and `invert_batch` in the other made it hard to DRY-out. Suggestions are welcome